### DISCUSSION TITLE
Games that you'd think have `.bsp`s, but don't

### DISCUSSION THREAD

**Discussion by snake-biscuits** [snake-biscuits/bsp_tool#115] [Show and tell]
* Call of Duty 3 (Xbox 360)
 * Quake (Nintendo 64)
 * Quake II (Nintendo 64)
 * Quake II (Sony PSX)

**Comment by kungfooman** [comment]
Interesting, so which model format is Quake 2 on Nintendo 64 using then?

**Reply by snake-biscuits** [reply]
AFAIK there's no way to interpret a N64 ROM as files
Scrolling through a hexdump I couldn't find anything resembling `.bsp`s either
Quake II PSX does have files, but maps are spread across a unique `.dat` format

**Reply by kungfooman** [reply]
Well, the ROM's could be encoded/compressed, that's why one usually loads the game in an emulator and then checks/searches the game RAM.

**Reply by snake-biscuits** [reply]
That's an idea, idk if I really want to go digging into it tho
Since #16 would force me to make & test custom maps on a physical N64

**Comment by snake-biscuits** [comment]
Found a little more info on Quake II's ports:
Digital Foundry on Youtube: [DF Retro: Quake 2 - The Legacy And The Ports!](https://www.youtube.com/watch?v=DFYjSkUdfb4)
Apparently the Quake II N64 version is built on Quake 64

Wasn't aware of the Xbox360 version, will look into that later
